pytest_deep_analysis/runtime/collectors.py

```
@@ -0,0 +1,194 @@
+                function_name=func_name,
+                module=module,
+                args=(),  # Simplified for now
+                kwargs={},
+                lineno=lineno
+            )
+            self.traces.append(trace)
+
+        # Capture assertions (look for assert statement execution)
+        # Note: Python doesn't have a direct "assert" event, but we can infer from AssertionError
+        elif event == "exception":
+            exc_type, exc_value, exc_tb = arg
```

Ruff (0.14.4)

Unpacked variable `exc_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)